src/main/java/org/jabref/gui/util/DynamicallyChangeableNode.java

```diff
@@ -0,0 +1,16 @@
+package org.jabref.gui.util;
+
+import javafx.scene.Node;
+import javafx.scene.layout.Priority;
+import javafx.scene.layout.VBox;
+
+/*
+ * A node that can change it's content using a setContent(Node node) method, similar to {@link Tab}.
+ */
+public class DynamicallyChangeableNode extends VBox {
+    protected void setContent(Node node) {
+        getChildren().clear();
+        VBox.setVgrow(node, Priority.ALWAYS);
```

Comment:
This should go to the constructor?

Reply:
Which constructor? No. That's the purpose of the component.

`getChildren().clear()` for clearing previous content.

`VBox.setVgrow(...` to expand the element. User don't have to set it, because that's an implementation detail. By implementation detail I mean, that the class could safely use an `HBox` instead of `VBox`, and then we need to call `HBox.setHgrow(...)`.